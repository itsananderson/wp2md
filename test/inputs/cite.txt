== Cite ==

This is a <cite>cite</cite> tag.

This is multiple <cite>cite</cite> tags on the same <cite>line</cite>
