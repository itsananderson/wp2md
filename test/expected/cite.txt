Cite
----

This is a *cite* tag.

This is multiple *cite* tags on the same *line*
